### DISCUSSION TITLE
Caddyfile

### DISCUSSION THREAD

**Discussion by kubo6472** [NamelessMC/Nameless#2287] [Q&A] [answered]
Is anyone successfully running Nameless 2.0.0 pr9 with friendly URLs (rewrites) on Caddy 2? I have every site in one caddyfile and I like it so far. But I can not seem to figure out, how to properly translate apache mod_rewrite to something caddy understands.

**Comment by Derkades** [comment, marked as answer]
I have never used caddy before, but according to the documentation it looks like this nginx config
```
location / {
    try_files $uri $uri/ /index.php?route=$uri&$args;
}
```
translates to 
```
try_files {path} {path}/ /index.php?route={path}&{query}
```
maybe? https://caddyserver.com/docs/caddyfile/directives/try_files

No idea how location blocks work in caddy but presumably you do. Next thing would be to deny access to all `.tpl` and `.cache` files.
